k9mail/src/main/java/com/fsck/k9/view/MessageHeader.java

```diff
@@ -154,11 +177,32 @@ private void onAddSenderToContacts() {
         }
     }
 
+    public String createMessage(int addressesCount){
+        return mContext.getResources().getQuantityString(R.plurals.copy_address_to_clipboard,addressesCount);
+    }
+
+    private void onAddAddressesToClipboard(Address[] addresses){
+        StringBuilder addressesToCopy = new StringBuilder();
+        for(Address addressTemp : addresses){
+            addressesToCopy.append(addressTemp.getAddress() + " ");
```

Comment:
I think it might be more useful to separate them with a `;` as that's the standard email separator.